scripts/build

```diff
@@ -7,3 +7,5 @@ echo Running: build-rancherd
 $(dirname $0)/build-rancherd
 echo Running: build-agent
 $(dirname $0)/build-agent
+echo Running: build-token-generator
+$(dirname $0)/../tests/v2/integration/scripts/build-integration-setup
```

Comment:
Build is meant for building the rancher binaries, its not a generic build area. That way devs can build only the binaries, or run only the tests. So this would be better in the ci or test bash areas.

Reply:
Updated